soh/CMakeLists.txt

```
@@ -1979,13 +1979,16 @@ endif()
 
 if(CMAKE_SYSTEM_NAME MATCHES "NintendoSwitch")
 
-set_target_properties(soh PROPERTIES
-        APP_TITLE "Ship of Harkirian"
-        APP_AUTHOR "Ship"
-        APP_VERSION "3.0.0"
-        ICON "icon.jpg")
-
-nx_create_nro(soh)
+nx_generate_nacp(Ship.nacp
+   NAME "Ship of Harkirian"
+   AUTHOR "Ship"
```

Should probably be this:
```suggestion
   AUTHOR "Harbour Masters"
```